source/manual/setting-up-new-rails-app.html.md

```diff
@@ -8,140 +8,161 @@ last_reviewed_on: 2018-03-22
+[govuk-puppet-jenkins]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata/common.yaml
+[dns]: https://docs.publishing.service.gov.uk/manual/dns.html#making-changes-to-publishingservicegovuk
+[govuk-saas-config]: https://github.com/alphagov/govuk-saas-config
+[govuk-app-deployment]: https://github.com/alphagov/govuk-app-deployment
+[sentry]: https://sentry.io/settings/govuk/teams/
+[release]: https://release.publishing.service.gov.uk/applications
+[deploy-jenkins]: https://deploy.integration.publishing.service.gov.uk/job/Deploy_App/
 
-- Your app follows GOV.UK conventions, including the [Ruby styleguide](https://gds-way.cloudapps.digital/manuals/programming-languages.html#code-styleguides)
-- Continuous integration is set up
-- Your app can be deployed to integration, staging and production environments
-- You can monitor how your app is performing
+## Bootstrapping a new application
 
-For most apps, this means you'll be using:
+Run the following to avoid some of the default rails boilerplate.
 
-- [Jenkins](https://jenkins.io) for CI
-- [Puppet](https://puppet.com/product) to provision the machines your app runs on
-- [Capistrano](http://capistranorb.com/) to deploy it
-- [Sentry](https://sentry.io/govuk) to track stack traces
-- The [GOV.UK development VM](https://github.com/alphagov/govuk-puppet/tree/master/development-vm)
+```
+rails new myapp --skip-javascript --skip-test --skip-bundle --skip-spring --skip-action-cable --skip-action-mailer --skip-active-storage
+```
 
-Exceptions are apps that don't need to share infrastructure with the rest of GOV.UK, such as internal tools or prototypes. In this case it might be easier to deploy to the [Government PaaS](https://docs.cloud.service.gov.uk/#technical-documentation-for-gov-uk-paas) or [Heroku](https://devcenter.heroku.com/articles/getting-started-with-ruby).
+Replace the Gemfile with the gems you need. Here is an example.
 
-This guide will assume you aren't doing that.
+```
+ruby File.read(".ruby-version").strip
 
-## Bootstrapping a new application
+source "https://rubygems.org"
 
-The quickest and easiest way to start a new Rails application on the
-GOV.UK stack is to use the [govuk-rails-app-template
-repository](https://github.com/alphagov/govuk-rails-app-template).
+gem "rails", "~> 5.2"
 
-This will create a new Rails application, installing and configuring common features.
+gem "bootsnap", "~> 1"
+gem "govuk_app_config", "~> 1"
+gem "pg", "~> 1"
+gem "plek", "~> 2"
 
-From here on we'll assume you’re configuring a new
-Rails application named `myapp` that was created with the above
-script.
+group :development do
+  gem "byebug"
+  gem "govuk-lint"
+  gem "listen"
+end
 
-## Naming conventions
+group :test do
+  gem "factory_bot_rails"
+  gem "faker"
+  gem "simplecov", require: false
+end
 
-There are a few conventions for naming apps in the Puppet configuration
-and elsewhere:
+group :development, :test do
+  gem "rspec-rails"
+end
+```
```

Comment:
Be good to get the versions on the dev/test ones. Might be worth just bringing this more up to date with content-publisher as this looks pre-review?